draft-ietf-scitt-architecture.md

```diff
@@ -373,280 +369,155 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
+For example, a Transparency Service that is supporting a software package management system, might be referenced from the APIs exposed for package management.
+Providing an ability to request a fresh receipt for a given software package, or to request a list of signed statements and artifacts associated with a software package.
 
-1. the CWT_Claims Issuer (or its resolved key manifest)
-1. the collection of Transparent Statements to which this Statement about the Artifact belongs (CWT_Claims Subject)
-1. the list of service identities of trusted Transparency Services
+## Signed Statements
 
-When presented with a Transparent Statement for an Artifact, Verifiers verify the CWT_Claims Issuer identity, signature, and Receipt.
-They may additionally apply a validation policy based on the protected headers present both in the Envelope, the Receipt, or the Statement itself, which may include security-critical or Artifact-specific details.
+This specification prioritizes conformance to RFC9052 and its required and optional properties.
+Profiles and implementation specific choices should be used to determine admissability of conforming messages.
+This specification is left intentionally open to allow implementatons to make the restrictions that make the most sense for their operational use cases.
 
-Some Verifiers may systematically fetch all Transparent Statements using the CWT_Claims Subject and assess them alongside the Transparent Statement they are verifying to ensure freshness, completeness of evidence, and Non-equivocation.
+At least one identifier for an identity document MUST be included in the protected header of the COSE envelope, either `x5t` or `kid`.
 
-Some Verifiers may choose to subset the collection of Statements, filtering on the payload type (Protected Header `3`), the CWT (Protected Header `15`) Issuer claim, or other non-opaque properties.
+Support for `x5t` is mandatory to implement.
 
-Some Verifiers may systematically resolve Issuer identifiers to fetch the latest corresponding verification keys.
-This behavior strictly enforces the revocation of compromised keys.
-Once the Issuer has updated its Statement to remove a key identifier, all Signed Statements include the corresponding `kid` will be rejected.
+Support for `kid` is optional.
 
-Some Verifiers may decide to skip the identifier-based signature verification, relying on the Transparency Service's Registration Policy and the scrutiny of other Verifiers.
-Although this weakens their guarantees against key revocation, or against a corrupt Transparency Services, they can still keep the Receipt and blame the Issuer or the Transparency Services at a later point.
+When `x5t` is present, `iss` MUST be a string with a value between 1 and 8192 characters in length that fits the regular expression of a distinguished name.
 
-# Signed Statement Issuance, Registration, and Verification
+TODO: address the guidance on choosing `iss` values for certificate based identities.
```

Comment:
That is a big TODO and since x5t is now mandatory it needs to be specified before this PR can be merged

Reply:
I would prefer release early, release often, The note should move to an issue, though.